db/test/test_data.py

```diff
@@ -263,6 +264,11 @@ def test_add_get_model(self):
 
     def test_get_summary_data(self):
         pass
+    
+    def test_check_low_level_duplicates(self):
```

Comment:
Or you can just write a test that fails and finish it with the rest of implementation.